charts/seed-monitoring/charts/grafana/dashboards/operators/kubernetes-pods-dashboard.json

```diff
@@ -5,8 +5,7 @@
   "editable": true,
   "gnetId": null,
   "graphTooltip": 1,
-  "id": 3,
```

Comment:
This should be there to have stable links?

Reply:
@wyb1 is on vacation until beginning of August. @mvladev: Can we merge this PR also in this state? If you don't agree, can you please push to his branch and adapt it accordingly so that we can merge it then?

Reply:
ping

Reply:
done.